about.html

```diff
@@ -0,0 +1,160 @@
+          <div  class="about-paragraph">
+            <p>
+              According to the World Bank, financial inclusion means individuals and businesses having access to basic financial services and products such as bank accounts, loans, and insurance, in a manner that is sustainable and affordable. When there is little or no access to these financial products or services, recourse is made to cash-based transactions – with its attendant risks – and savings are done through informal means, such as esusu, ajo or the local thrift collectors.
+            </p>
+          </div>
+        </div>
+        <div class="reach-out">
+          <div class="paragraph-reach-out">
+            <p>Please you reach out to us via Email incase you want to find out more about DMT summit 2019!</p>
+          </div>
+          <p>taiwoenochdev@gmail.com</p>
+
+          <div class="bottom-line"></div>
+
+        </div>
+      </section>
+      <section>
+        <div class="summit-div">
+          <div class="creative">
+            <p>Danmert Nigeria Enterprise Summit 2019 Logo</p>
+          </div>
+
+          <div class="bottom-line-2"></div>
+
+          <div class="logo-description">
+            <p>The logo of DMT Summit was decided through the logo competition from 8. june to 7. july.</p>
+          </div>
+          <div>
+            <img src="./images/main-about-logo.png" alt="summit-logo"  class="summit-logo">
+          </div>
+        </div>
+        <div class="towards-end">
+          <div class="about-summit">
+            <p>See the past DMT Summits</p>
+
+            <div class="bottom-line-2"></div>
+
+          </div>
+          <div class="towards-end-paragraph">
+            <p>Take a look at last two Danmert Enterprise summit that took place in Nigeria</p>
+          </div>
+          <div class="need-flex">
+            <div class="summit-2013">
+              <p>2018 DMT Submit</p>
+            </div>
+            <div class="summit-2014">
+              <p>2017 DMT Submit</p>
+            </div>
+          </div>
+        </div>
+      </section>
+    </header>
+    <section>
+      <div class="site-icons">
+        <div>
+          <p class="partner">Partner</p>
+        </div>
+
+        <div class="border"></div>
+
+        <div class="bottom-icons">
+          <div>
+            <img src="./images/mozilla.png" alt="mozilla-icon" class="partner-1">
+          </div>
+          <div>
+            <img src="./images/google.png" alt="google-icon" class="partner-1">
+          </div>
+          <div>
+            <img src="./images/naver.png" alt="naver-icon" class="partner-1">
+          </div>
+          <div class="bottom-icon-2">
+            <div>
+              <img src="./images/kakao1.png" alt="kakao-icon" class="partner-1">
+            </div>
+            <div>
+              <img src="./images/airbnb.png" alt="airbnb-icon" class="partner-1">
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] Kindly remove the _**Partners**_ section from the _**desktop**_ view of the `About` page. Based on the original design, this section is present in the **_mobile_** view but not the _**desktop**_ view. Kindly implement this change. 👍 💻

Reply:
Done